examples/basics/plotting/quick.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Example of plotting multiple things using the quick interface.
+"""
+
+import numpy as np
+
+from vispy.io import load_data_file
+import vispy.plot.q as vpq
```

Comment:
I don't think the "quick" plotting methods should live in the same namespace as the figure-based methods. Users of other plotting libraries e.g. matplotlib or MATLAB will be confused about the differences in behavior, e.g. why their plotting functions return figures/canvases instead of a visual, and why calling `plot` more than once causes multiple windows to show up.

I am -1 on having an entirely different top-level package, either, but I can live with it if other people want it.